packages/website/src/features/Debug/DebugPage.tsx

```diff
@@ -28,93 +36,111 @@ export const DebugPage = () => {
-          deployments, it can also assist with errors decoding of unknown data.
-          This guide will show a few examples of how to use Cannon to explain
-          errors, get stack traces, and gather other sorts of useful data you
-          would want when helping a user. The best part: Cannon does this with{' '}
-          <i>no centralized data providers</i>.
+          Protocol development can often involve frustrating errors that consist
+          of inscrutable bytecode. Function calls staged to a Safe might not be
+          legible. A dreaded <Code>0x</Code> error may be returned from a
+          contract with no reason for the revert to be found.
         </Text>
-        <Heading size="md" mb={4} mt={6}>
-          Decoding any Data
-        </Heading>
         <Text mb={4}>
-          If you havent already, you should install/upgrade Cannon:
+          Cannon can leverage data from packages to decode bytecode, generate
+          human-readable stack traces, and send transactions to protocols.
+        </Text>
+        <Text mb={2}>
+          To use the <Code>decode</Code>, <Code>trace</Code>, and{' '}
+          <Code>interact</Code> commands, start by installing/upgrading Cannon:
         </Text>
         <Box mb={4}>
           <CommandPreview command="npm i -g @usecannon/cli" />
         </Box>
+        <Heading size="md" mb={4} mt={6}>
+          Decode
+        </Heading>
         <Text mb={4}>
-          For this guide we will use the{' '}
-          <Link href="https://usecannon.com/packages/synthetix">synthetix</Link>
-          package as an example. Of course, you can substitute these commands
-          for any other package.
+          You may encounter a hex string related to a protocol, but can’t tell
+          what it is. Centralized services such as{' '}
+          <Link isExternal href="https://openchain.xyz/tools/abi">
+            ABI Tools
+          </Link>{' '}
+          or the{' '}
+          <Link isExternal href="https://www.4byte.directory/">
+            Ethereum Signature Database
+          </Link>{' '}
+          may be able to help if the relevant ABI has been uploaded there, but
+          won’t be useful during protocol development.
         </Text>
         <Text mb={4}>
-          Lets have cannon explain what the hex `0x` means. To do this, we can
-          use the `cannon decode` utility. For exmaple:
+          You can pass hex data to Cannon’s <Code>decode</Code> command, along
+          with the package name and a relevant chain ID, to get a human-readable
+          version of function calls, function output, event data, and error
+          data.
         </Text>
+        <Text mb={2}>For example, you can decode this hex data ... :</Text>
         <Box mb={4}>
-          <CommandPreview command="cannon decode synthetix 0x" />
-          Command Output
+          <CommandPreview command="cannon decode TODO-EXAMPLE" />
```

Comment:
`cannon decode synthetix-omnibus --chain-id 84531 --preset competition 0xb87daa32000000000000000000000000000000000000000000000000000000006502188b00000000000000000000000000000000000000000000000000000000650218190000000000000000000000000000000000000000000000000000000065021855`

Reply:
can you also show the ? many people reading this guide will not run the command themselves on their computer, but they should be able to see what the output should be.

Reply:
![image](https://github.com/usecannon/cannon/assets/2107457/61385792-917c-47c3-b143-ba13735fc994)